jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js

```diff
@@ -104,12 +104,22 @@ YUI.add('deployment-add-credentials', function() {
           <juju.components.DeploymentPanelContent
             title="Configure Amazon Web Services">
             <form>
+              <label className="deployment-panel__label"
+                htmlFor="credential-name">
+                Credential name
+              </label>
               <input className="deployment-panel__input"
-                placeholder="Credential name"
+                id="credential-name"
+                placeholder="AWS_1"
                 type="text"
                 ref="templateName" />
+              <label className="deployment-panel__label"
+                htmlFor="specify-region">
+                Specify region
+              </label>
               <input className="deployment-panel__input"
-                placeholder="Specify region"
+                id="specify-region"
+                placeholder="us-central1"
                 type="text"
```

Comment:
Yes it will be some point in the future, we still have a bit of backend work to do before that's a possibility.